src/models/users.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const Users = sequelize.define('Users', {
+    firstName: DataTypes.STRING,
+    lastName: DataTypes.STRING,
+    username: DataTypes.STRING,
+    email: DataTypes.STRING,
+    password: DataTypes.STRING,
+    role: DataTypes.BOOLEAN,
+    isVerified: DataTypes.BOOLEAN
+  }, {});
+  Users.associate = function(models) {
```

Comment:
This will give Eslint error. You can remove the associate as we are having none now.